main.c

```diff
@@ -40,36 +45,38 @@ void insertNode (int position ,int item, struct node **ptrtohead)
 int main()
 {
     // Create Linked list
-    struct node *head= NULL, *temp, *constanthead;

```

Comment:
I changed the name `head` to `current` because it represents my mental model of the node I am "currently holding in my hands", which is usually the most-recently added node.